test/fast/Unit tests/nvm_supports_xz

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+cleanup () {
+  rm ../../xz-test/{which,command,awk,rm}
+  rmdir ../../xz-test
+}
+die () { echo $@ ; exit 1; }
+
+. ../../../nvm.sh
+
+mkdir ../../xz-test
+touch ../../xz-test/xz
+chmod +x ../../xz-test/xz
+ln -s /usr/bin/which ../../xz-test/which
+ln -s /usr/bin/command ../../xz-test/command
+ln -s /usr/bin/awk ../../xz-test/awk
+ln -s $(which rm) ../../xz-test/rm
+
+
+OLDPATH=$PATH
+export PATH=../../xz-test
```

Comment:
I previously had export PATH=$OLDPATH but since every test seems sourced separately I didn't see anything breaking. Happy to re-add? If I prepend dir we will very likely run into false positives since most machines we will be testing on already has `xz` installed.